pkg/api/query/grpc.go

```diff
@@ -168,6 +169,7 @@ func (g *GRPCAPI) QueryRange(request *querypb.QueryRangeRequest, srv querypb.Que
 		request.EnableQueryPushdown,
 		false,
 		request.ShardInfo,
+		query.NoopSeriesStatsReporter,
```

Comment:
why is this set to no-op here?

Reply:
Because this is an internally initiated request, not something started by a user.